developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:checksums />              | A list of checksum.                              | ``md5:04c36b75222cd9fd47f2607333029106``                                 |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <nc:has-preview />            | Whether a preview of the file is available.      | ``true`` or ``false``                                                    |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:size />                   | | Unlike ``getcontentlength``,                   |                                                                          |
+|                               | | this property also works                       |                                                                          |
+|                               | | for folders, reporting the size                |                                                                          |
+|                               | | of everything in the folder.                   |                                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:quota-used-bytes />       | Amount of bytes used in the folder.              | ``3950773``                                                              |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:quota-available-bytes />  | Amount of available bytes in the folder.         | | ``3950773``                                                            |
+|                               |                                                  | | ``-1`` Uncomputed free space.                                          |
+|                               |                                                  | | ``-2`` Unknown free space.                                             |
+|                               |                                                  | | ``-3`` Unlimited free space.                                           |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <nc:rich-workspace-file />    | The workspace file.                              |                                                                          |
```

Comment:
It's the id of the workspace file: https://github.com/nextcloud/text/blob/5c4bac7e2305be8632b316ca61fc66c2d43805c1/lib/DAV/WorkspacePlugin.php#L125